src/script/conversation/ConversationRepository.ts

```diff
@@ -872,17 +872,27 @@ export class ConversationRepository {
    * Check for conversation locally and fetch it from the server otherwise.
    * TODO(Federation): Remove "optional" from "domain"
    */
-  async getConversationById(conversation_id: QualifiedId): Promise<Conversation> {
+  async getConversationById(conversation_id: QualifiedId, searchInLocalDB = false): Promise<Conversation> {
```

Comment:
direct search in indexedDB is hidden under optional param, so we don't do it for all the cases